src/Controller/MultiStepController.php

```diff
@@ -28,27 +29,40 @@ class MultiStepController extends Controller
 
     /** @var ArgumentResolverInterface */
     private $argumentResolver;
+    /**
+     * @var FlowContextFactoryInterface
+     */
+    private $flowContextFactory;
+    /**
```

Comment:
For my personal taste one single empty line between a member variable declaration and the following annotation comment should be inserted to better differentiate visually.